instrumentation/play/play-mvc/play-mvc-2.4/javaagent/src/play24Test/java/io/opentelemetry/javaagent/instrumentation/play/v2_4/client/PlayWsClientTest.java

```diff
@@ -78,6 +78,9 @@ protected void configure(HttpClientTestOptions.Builder optionsBuilder) {
           return attributes;
         });
 
+    optionsBuilder.setTestConnectionFailure(false);
+    optionsBuilder.setSpanEndsAfterType(null);
```

Comment:
spanEndsAfterHeaderReceived is a little bit over 2 seconds, i..e  `2.0509351`

Reply:
Try adding `optionsBuilder.spanEndsAfterBody();`